src/layouts/SiteLaunchPad/components/SiteLaunchInfoGathering.tsx

```diff
@@ -60,50 +71,86 @@ export const SiteLaunchInfoCollectorBody = ({
     })
     setPageNumber(SITE_LAUNCH_PAGES.RISK_ACCEPTANCE)
   }
-  const [selectedRadio, setSelectedRadio] = useState("")
+  const [selectedDomainNature, setSelectedDomainNature] = useState("")
+  const [wwwSetting, setWwwSetting] = useState("")
+  const [displayWwwQn, setDisplayWwwQn] = useState(false)
+  const [isWwwRecommended, setIsWwwRecommended] = useState(true)
+
+  const handleDomainNatureQn = (response: string) => {
+    if (response === "") {
+      // keep the state as is, disallow unselecting
+      return
+    }
+    setIsWwwRecommended(recommendWwwValue(watch("domain")))
```

Comment:
`setIsWwwRecommended` is being done in the useeffect right, is this still required here?

Reply:
thanks for the catch, fixed here 3f489afb5c0bb2674d10aba6aead6a8f2c36bcbd